jerry-core/ecma/operations/ecma-typedarray-object.c

```diff
@@ -303,6 +312,12 @@ ecma_typedarray_create_object_with_typedarray (ecma_object_t *typedarray_p, /**<
   else
   {
     ecma_length_t byte_length = array_length << element_size_shift;
+
```

Comment:
Question: byte_length = array_length << element_size_shift this shift could also lead to overflow (resulting a 0 length).